### PR TITLE
feat: inputRichText支持change监听事件

### PR DESCRIPTION
**Pull request by nabaonan**
### <samp>🤖 Generated by Copilot at e50c0d7</samp>

This pull request enhances the `InputRichText` component by adding event support and documentation. The component can now emit a `change` event with the rich text value, which can be used by other components or actions.

### <samp>🤖 Generated by Copilot at e50c0d7</samp>

> _If you want to edit rich text_
> _And react to changes with logic_
> _Use `InputRichText`_
> _And listen to `change` event_
> _You can access the data with a magic syntax_

### Why

修复issue中 需求，    给inputRichText添加一个change事件用来监听内容变化
#7911 

### <samp>🤖 Generated by Copilot at e50c0d7</samp>

*  Add a new section to the documentation of the InputRichText component, explaining the events it emits and how to use actions based on the event data ([link](https://github.com/baidu/amis/pull/8011/files?diff=unified&w=0#diff-5763a9c8be2add40d5fb4ab942d81bd41c7206e75257284d7fbac17f9248cec8R233-R242))
*  Import `resolveEventData` function to resolve the event data based on the component props and the event payload, supporting expressions and filters ([link](https://github.com/baidu/amis/pull/8011/files?diff=unified&w=0#diff-e1350ae9ec131bde1820e064b1d18297f11fd57e531554661d21d233dfbe1a15R8))
*  Import `debounce` function to create a debounced version of the `dispatchEvent` method, preventing firing the event too frequently ([link](https://github.com/baidu/amis/pull/8011/files?diff=unified&w=0#diff-e1350ae9ec131bde1820e064b1d18297f11fd57e531554661d21d233dfbe1a15R15))
*  Define `RichTextRendererEvent` type alias to represent the possible event names that the InputRichText component can emit, currently only `change` ([link](https://github.com/baidu/amis/pull/8011/files?diff=unified&w=0#diff-e1350ae9ec131bde1820e064b1d18297f11fd57e531554661d21d233dfbe1a15R66-R67))
*  Call `dispatchEvent` method with the `change` event name and the current rich text value as the event data, whenever the `onChange` callback of the editor is triggered ([link](https://github.com/baidu/amis/pull/8011/files?diff=unified&w=0#diff-e1350ae9ec131bde1820e064b1d18297f11fd57e531554661d21d233dfbe1a15R271))
*  Define `dispatchEvent` method as a debounced function that uses the `dispatchEvent` prop and the `resolveEventData` function to dispatch the event to the outside listeners, with a debounce interval of 250 milliseconds ([link](https://github.com/baidu/amis/pull/8011/files?diff=unified&w=0#diff-e1350ae9ec131bde1820e064b1d18297f11fd57e531554661d21d233dfbe1a15R280-R297))

### CODE REVIEW

File: packages/amis/src/renderers/Form/InputRichText.tsx [hunk 1]
```diff
@@ -272,6 +277,24 @@ export default class RichTextControl extends React.Component<
     return env?.loadTinymcePlugin?.(tinymce);
   }
 
+  dispatchEvent = debounce(
```
Comment (2betop) [inline packages/amis/src/renderers/Form/InputRichText.tsx:280]: debounce 方法，组件销毁的时候需要 clear

**Review by hsm-lv — CHANGES_REQUESTED** [review]

File: packages/amis/src/renderers/Form/InputRichText.tsx (hunk 1 above)
Comment (nabaonan) [reply packages/amis/src/renderers/Form/InputRichText.tsx:280]: > 动作执行本身就支持debounce

试了，如果不加防抖，频繁触发

Comment (hsm-lv) [reply packages/amis/src/renderers/Form/InputRichText.tsx:280]: 我的意思是不用管组件本身的事件是否频繁派发，只需要关心事件动作那里能不能通过防抖配置解决你的问题就可以了

**Comment by nabaonan** [comment]
已有相同的pr了 #8433

**Pull request closed without merging**